code/modules/religion/rites.dm

```diff
@@ -255,17 +255,94 @@
 
 // Necro Rites
 
+/datum/religion_rites/create_lesser_lich
+	name = "Create Lesser Lich"
+	desc = "Gives the bound creature a spell granting them the ability to create a lesser phylactery, causing them to become a skeleton and revive on death twice if the phylactery still exists on-station. Be warned, becoming a lesser lich will prevent revivial by any other means."
+	ritual_length = 70 SECONDS
+	ritual_invocations = list("From the depths of the soul pool ...",
+	"... come forth into this being ...",
+	"... grant this servant power ...",
+	"... grant them temporary immortality ...")
+	invoke_msg = "... Grant them the power to become one with necromancy!!"
+	favor_cost = 2250
+//the creature chosen for the rite
```

Comment:
the comments should be indented and formatted like the following:
```
/// Comment here
```
The three slashes are necessary. Also, describe more ("the spell" -> "The typepath of the spell to grant")